R/download.R

```diff
@@ -38,11 +39,17 @@ download_hydat <- function(dl_hydat_here = NULL) {
     }
   }
 
-  ans <- ask(paste("Downloading HYDAT will take ~10 minutes.","This will remove any older versions of HYDAT",
-                   "Is that okay?", sep = "\n"))
+  if (!is.logical(ask)) stop("Parameter ask must be a boolean")
+  
+  if (ask) {
+    ans <- ask(paste("Downloading HYDAT will take ~10 minutes.","This will remove any older versions of HYDAT",
+                     "Is that okay?", sep = "\n"))
+  } else {
+    ans <- TRUE
```

Comment:
I think some chattiness would be good here. If included as part of a function one can always suppress the message.
```suggestion
    ans <- TRUE
    green_message(paste0("Downloading HYDAT to ", normalizePath(dl_hydat_here)))
```